src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -13,6 +13,7 @@
 
 var ReactElement = require('ReactElement');
 var ReactPropTypeLocationNames = require('ReactPropTypeLocationNames');
+var warning = require('warning');
```

Comment:
Please move this require under `getIteratorFn` (style isn't totally obvious here, sorry)